Maestro/subscriptions.json

```diff
@@ -556,66 +556,24 @@
         }
       }
     },
-    // Update cli master dependencies in dotnet-docker-nightly master
+    // Update dependencies in dotnet-docker-nightly master
     {
       "triggerPaths": [
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/master/Latest_Packages.txt"
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.1/installers.semaphore"
       ],
       "action": "dotnet-docker-nightly-general",
       "delay": "00:05:00",
       "actionArguments": {
         "vsoBuildParameters": {
-          "ScriptFileName": "update-dependencies\\update-dependencies.ps1",
+          "ScriptFileName": "update-dependencies\\Invoke-UpdateDependencies.ps1",
           "Arguments": [
-            "-EnvVars",
-            "'GITHUB_USER=dotnet-maestro-bot',",
-            "'GITHUB_EMAIL=dotnet-maestro-bot@microsoft.com',",
-            "'GITHUB_PASSWORD=`$(`$Secrets[`'DotNetMaestroBotGitHubToken`'])',",
-            "'CLI_BRANCH=master',",
-            "'CLI_RELEASE_PREFIX=2.1.0',",
-            "'RUNTIME_RELEASE_PREFIX=2.1.300'"
-          ]
-        }
-      }
-    },
-    // Update cli release/2.0.0 dependencies in dotnet-docker-nightly master
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/release/2.0.0/Latest_Packages.txt"
-      ],
-      "action": "dotnet-docker-nightly-general",
-      "delay": "00:05:00",
-      "actionArguments": {
-        "vsoBuildParameters": {
-          "ScriptFileName": "update-dependencies\\update-dependencies.ps1",
-          "Arguments": [
-            "-EnvVars",
-            "'GITHUB_USER=dotnet-maestro-bot',",
-            "'GITHUB_EMAIL=dotnet-maestro-bot@microsoft.com',",
-            "'GITHUB_PASSWORD=`$(`$Secrets[`'DotNetMaestroBotGitHubToken`'])',",
-            "'CLI_BRANCH=release/2.0.0',",
-            "'CLI_RELEASE_PREFIX=2.0.3'"
-          ]
-        }
-      }
-    },
-    // Update cli rel/1.1.0 dependencies in dotnet-docker-nightly master
-    {
-      "triggerPaths": [
-        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/cli/rel/1.1.0/Latest_Packages.txt"
-      ],
-      "action": "dotnet-docker-nightly-general",
-      "delay": "00:05:00",
-      "actionArguments": {
-        "vsoBuildParameters": {
-          "ScriptFileName": "update-dependencies\\update-dependencies.ps1",
-          "Arguments": [
-            "-EnvVars",
-            "'GITHUB_USER=dotnet-maestro-bot',",
-            "'GITHUB_EMAIL=dotnet-maestro-bot@microsoft.com',",
-            "'GITHUB_PASSWORD=`$(`$Secrets[`'DotNetMaestroBotGitHubToken`'])',",
-            "'CLI_BRANCH=rel/1.1.0',",
-            "'CLI_RELEASE_PREFIX=1.1.0'"
+            "-CleanupDocker",
+            "-UpdateDependenciesParams '",
+              "--user dotnet-maestro-bot",
```

Comment:
Do these need spaces after the args?

(Btw, I like the indentation.)

Reply:
yes - I am expecting this is the format of the `Arguments` passed to the build def - `-CleanupDocker -UpdateDependenciesParams ' --user dotnet-maestro-bot --email dotnet-maestro-bot@microsoft.com --password `$(`$Secrets[`'DotNetMaestroBotGitHubToken`']) https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/product/cli/release/2.1/build.xml '`

Am I missing something?

Reply:
Hah, sorry, I just forgot that these args are joined by ` `, not concatenated. My bad. :smile: